README.md

```diff
@@ -506,6 +508,11 @@ You can see the bundle in action in the [custom libp2p example](examples/custom-
         - `kBucketSize` (number): bucket size (default `20`)
         - `randomWalk` (object): configuration for random walk
             - `enabled` (boolean): whether random DHT walking is enabled (default `false`)
+    - `pubsub` (object): Configuration options for Pubsub
+        - `enabled` (boolean): if pubbsub subsystem should be enabled (default: `true`)
+        - `emitSelf` (boolean): whether the node should emit to self on publish, in the event of the topic being subscribed (default: `true`)
```

Comment:
@alanshaw we are on the process of probably defaulting this to `false` on libp2p side [libp2p/js-libp2p-pubsub#22](https://github.com/libp2p/js-libp2p-pubsub/issues/22). There are tests who expect this to be `true` in `js-ipfs`. 

Anyway, this change will only occur in a next breaking release of `js-libp2p`. If by then, you would like to change this to `false` in ipfs, let me know

Reply:
@vasco-santos we should explicitly set the js-ipfs config to `true` for emitSelf now, it's not doing that yet. That way if that changes in a breaking release, js-ipfs won't need an update until it wants to change.